dev/arkdark/arktype.tmLanguage.json

```diff
@@ -23,6 +26,21 @@
                 }
             ]
         },
+        "arktypeHelpers": {
+            "contentName": "meta.embedded.arktype.helper",
+            "begin": "\\.(morph|and|or)\\(",
```

Comment:
Is there a reason these can't be defined in the same place as `type` and `scope`? Their behavior should be the same